torch_geometric/explain/explanation.py

```diff
@@ -245,6 +245,53 @@ def visualize_feature_importance(
 
         return ax
 
+    def visualize_subgraph(self, node_index: int) -> str:
```

Comment:
```suggestion
    def visualize_subgraph(self) -> str:
```
`node_index` will be part of `Explanation`. https://github.com/pyg-team/pytorch_geometric/blob/f622fee0d1ab71d1b2f97f9cc31d7990b45d99d9/torch_geometric/explain/explainer.py#L189